core/templates/components/summary-tile/exploration-summary-tile.component.html

```diff
@@ -1,135 +1,193 @@
+                </span>
+                <span *ngIf="avgRating">
+                  {{ avgRating | number:'1.1-1' }}
+                </span>
+                <span *ngIf="!avgRating" [innerHTML]="'I18N_LIBRARY_N/A' | translate" class="oppia-inactive-summary-tile-attribute">
+                </span>
               </span>
-              <span *ngIf="numViews == '1'"  class="views-text">view</span>
-              <span *ngIf="numViews != '1'" class="views-text">views</span>
             </li>
             <li>
-              <span>
-                <i class="fas fa-circle bullet-separator"></i>
+              <span class="far fa-eye" [ngbTooltip]="'I18N_LIBRARY_VIEWS_TOOLTIP' | translate" placement="top" container="body">
+                <span class="oppia-icon-accessibility-label" [innerHTML]="'I18N_LIBRARY_VIEWS_TOOLTIP' | translate"></span>
+              </span>
+              <span *ngIf="numViews">
+                {{ numViews | summarizeNonnegativeNumber }}
+              </span>
+              <span *ngIf="!numViews" [innerHTML]="'I18N_LIBRARY_N/A' | translate" class="oppia-inactive-summary-tile-attribute">
               </span>
             </li>
             <li>
               <span *ngIf="lastUpdatedDateTime">
                 <span class="oppia-icon-accessibility-label">Last Updated</span>
-                {{ relativeLastUpdatedDateTime }}
+                {{ lastUpdatedDateTime }}
               </span>
               <span *ngIf="!lastUpdatedDateTime" [innerHTML]="'I18N_LIBRARY_N/A' | translate" class="oppia-inactive-summary-tile-attribute">
               </span>
             </li>
           </ul>
+          <button *ngIf="isCollectionPreviewTile" class="oppia-learner-confirm-button oppia-play-exploration-button mat-button  mat-default-theme e2e-test-play-exploration-button">
+            <span [innerHTML]="'I18N_PLAYER_PLAY_EXPLORATION' | translate"></span>
+          </button>
+        </div>
+      </a>
+    </mat-card>
+    <button mat-button class="oppia-learner-confirm-button oppia-return-to-parent-button e2e-test-return-to-parent-button"
+              *ngIf="isRefresherExploration" (click)="loadParentExploration()">
+      <span translate="I18N_PLAYER_RETURN_TO_PARENT"></span>
+      <i class="fas fa-arrow-right oppia-arrow-icon">
+      </i>
+    </button>
+  </div>
+  <div *ngIf="mobileCardToBeShown">
+    <oppia-learner-dashboard-icons *ngIf="showLearnerDashboardIconsIfPossible && userIsLoggedIn"
+                  [activityType]="activityType"
+                  [activityId]="explorationId"
+                  [activityTitle]="explorationTitle"
+                  [isContainerNarrow]="isContainerNarrow"
+                  [isAddToPlaylistIconShown]="!isOwnedByCurrentUser">
+    </oppia-learner-dashboard-icons>
+    <mat-card class="mobile-activity-summary-card" *ngIf="!isRefresherExploration" [ngStyle]="{'margin': '5px'}">
```

Comment:
Here and below, for ngStyle that are not dynamically changing CSS, can we define CSS classes and use those instead?

Reply:
Already written(https://github.com/SubhashThenua/oppia/blob/develop/core/templates/components/summary-tile/exploration-summary-tile.component.html#L92), but I have changed